specification/servicebus/resource-manager/Microsoft.ServiceBus/stable/2017-04-01/servicebus.json

```diff
@@ -1145,6 +1145,264 @@
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ServiceBus/namespaces/{namespaceName}/migrationConfigurations/{configName}": {
+      "put": {
+        "tags": [
+          "MigrationConfigs"
+        ],
+        "operationId": "MigrationConfigs_CreateAndStartMigration",
+        "x-ms-examples": {
+          "MigrationConfigurationsStartMigration": { "$ref": "./examples/Migrationconfigurations/SBMigrationconfigurationCreateAndStartMigration.json" }
+        },
+        "description": "Creates Migration configuration and starts migration of enties from Standard to Premium namespace",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/namespaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/configNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/MigrationConfigProperties"
+            },
+            "description": "Parameters required to create Migration Configuration"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Migration Config successfully created",
+            "schema": {
+              "$ref": "#/definitions/MigrationConfigProperties"
+            }
+          },
+          "201": {
+            "description": "Migration Config creation request received"
+          },
+          "default": {
+            "description": "ServiceBus error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "delete": {
+        "tags": [
+          "MigrationConfigs"
+        ],
+        "operationId": "MigrationConfigs_Delete",
+        "x-ms-examples": {
+          "MigrationConfigurationsDelete": { "$ref": "./examples/Migrationconfigurations/SBMigrationconfigurationDelete.json" }
+        },
+        "description": "Deletes a MigrationConfiguration",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/namespaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/configNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Delete Migration Config request accepted"
```

Comment:
I think you should also handle 204: not found the same way as 200

Reply:
Otherwise callers will get Exception in C# and Java which is undesired.

Reply:
resolved